### PR TITLE
Add support for words with <\>, <">, <'>, <SPC> (#9)

### PR DESCRIPTION
**Pull request by kamyabzad**

### CODE REVIEW

File: dictionary_client/commands.py [hunk 1]
```diff
@@ -32,11 +34,13 @@ def disconnect_command():
 
 @dictionary_server_command
 def define_word_command(word, db="*"):
-    return f"DEFINE {db} {word}"
+    word = format_word(word)
+    return f"Define {db} {word}"
```
Comment (jams2) [inline dictionary_client/commands.py]: For consistency, can we keep the command "DEFINE" in uppercase? e.g. https://datatracker.ietf.org/doc/html/rfc2229#section-3.2

File: dictionary_client/utils.py [hunk 2]
```diff
@@ -0,0 +1,25 @@
+import re
```
Comment (jams2) [inline dictionary_client/utils.py]: Could you add a module level docstring pointing to the grammar ( https://datatracker.ietf.org/doc/html/rfc2229#section-2.2 )? I think that will help with grokking the regular expressions.

File: dictionary_client/commands.py [hunk 3]
```diff
@@ -32,11 +34,13 @@ def disconnect_command():
...
 
 
 @dictionary_server_command
 def match_command(word, db="*", strategy="exact"):
+    word = format_word(word)
     return f"MATCH {db} {strategy} {word}"
```
Comment (jams2) [inline dictionary_client/commands.py]: A thought - maybe we could add a `Word` class, move the `format_word` helper onto a method there, and call it in `Word.__str__`. I think the way you've implemented this is great though, so I'll leave that to your taste.

**Review by jams2 — CHANGES_REQUESTED** [review]
Hey @kamyabzad - this is really great, thanks for the contribution! I've left a couple of small comments to address.

**Comment by jams2** [comment]
Fixes #9

File: dictionary_client/commands.py (hunk 1 above)
Comment (kamyabzad) [reply dictionary_client/commands.py]: Oh that's a typo, I'll fix it!

File: dictionary_client/utils.py (hunk 2 above)
Comment (kamyabzad) [reply dictionary_client/utils.py]: Added!

File: dictionary_client/commands.py (hunk 3 above)
Comment (kamyabzad) [reply dictionary_client/commands.py]: I like this more

**Comment by jams2** [comment]
@kamyabzad are you happy to be listed on contributors as  Kamyab Taghizadeh ?

**Comment by kamyabzad** [comment]
@jams2 That would be awesome! Thank you.

**Pull request merged**